docs/advanced/settings.md

```diff
@@ -539,61 +532,95 @@ When a host is unavailable or is powered off, the VM only reboots and does not m
 }
 ```
 
+### `volume-snapshot-class`
+
+**Definition**: VolumeSnapshotClassName for the VolumeSnapshot and VolumeSnapshotContent when restoring a VM to a namespace that does not contain the source VM.
+
+**Default value**: `longhorn`
+
+**Example**:
+
+`longhorn`
+
+---
+<p>&nbsp;</p>
+
 ## UI Settings
 
 ### `branding`
 
-_Available as of v1.2.0_
+**Versions**: v1.2.0 and later
 
-This setting allows you to globally re-brand the UI by customizing the Harvester product name, logos, and color scheme.
+**Definition**: Setting allows you to globally rebrand the Harvester UI by customizing the product name, logos, and color scheme.
 
-Default: **Harvester**
+**Default value**: **Harvester**
 
 ![containerd-registry](/img/v1.2/advanced/branding.png)
 
-You can set the following options and values:
+**Supported options and values**:
 
-- **Private Label:** This option replaces "Harvester" with the value you provide in most places.
-- **Logo:** Upload light and dark logos to replace the Harvester logo in the top-level navigation header.
-- **Favicon:** Upload an icon to replace the Harvester favicon in the browser tab.
-- **Primary Color:** You can override the primary color used throughout the UI with a custom color of your choice.
-- **Link Color:** You can override the link color used throughout the UI with a custom color of your choice.
+- **Private Label**: Product name or other text that replaces "Harvester" in most locations on the Harvester UI.
+- **Logo**: Logo image in the top-level navigation header. You must upload logos for both light and dark modes.
+- **Favicon**: Small image displayed next to the page title in the browser tab.
+- **Primary Color**: Main color used throughout the Harvester UI.
+- **Link Color**: Color used for link text throughout the Harvester UI.
 
 ### `ui-index`
 
-This setting allows you to configure the HTML index location for the UI.
+**Definition**: HTML index location for the Harvester UI.
 
-Default: `https://releases.rancher.com/harvester-ui/dashboard/latest/index.html`
+**Default value**: `https://releases.rancher.com/harvester-ui/dashboard/latest/index.html`
 
-#### Example
+**Example**:
 
 ```
 https://your.static.dashboard-ui/index.html
 ```
 
+### `ui-path`
+
+**Definition**: Path that describes the location of `index.html`, which is used to access the Harvester UI.
+
+`ui-path` serves as the entry point to the Harvester UI and is active only in the following situations:
+
+- The value of `ui-source` is `bundled`.
+- The value of `ui-source` is `auto`, but `ui-index` is unable to retrieve the HTML file.
+
+**Default value**: `/usr/share/harvester/harvester`
+
+**Examples**:
+
+`index.html` is stored in a container in `/home/samplefolder`. The value of `ui-source` is `bundled`.
+
+Situation 1: The value of `ui-path` is `/home/samplefolder`. When you access the Harvester UI, the content of .
```

Comment:
Just curious, is there any missing words?

Reply:
@Yu-Jack Sorry about that. I have added the missing words. [Preview](https://66949ae72cfa68e0dbd8e3ba--harvester-preview.netlify.app/v1.4/advanced/index#ui-path)